engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/bulk-actions-subscriptions.html

```diff
@@ -1,40 +1,84 @@
 <span page-title>{{ 'Content Host Bulk Subscriptions' | translate }}</span>
 
-<div class="details">
-  <section>
-    <h4 translate>Content Host Subscriptions</h4>
+<div bst-alert="info">
+  <p translate>
+    This action will affect only those Content Hosts that require a change.
+    If the Content Host does not have the subscription it will not be removed, for example.
```

Comment:
Looks good to me! If I am being super nitpicky I would rephrase the second sentence "For example, if the Content Host does not have the selected subscription, it will not be removed.", but I will leave that up to you.